static/scss/top-app-bar.scss

```diff
@@ -53,7 +53,25 @@
   width: 154px;
   height: auto;
 }
+.navbar-brand {
+  position: relative;
+  .mit-link, .xpro-link {
```

Comment:
Can we isolate only the common properties (`position`, `z-index` and `opacity`) into this and have a separate `.mit-link` and `.xpro-link` to explicitly specify `left`, `top`, `width` and `height`? That would make it more readable.